drivers/src/doe.rs

```diff
@@ -79,7 +79,7 @@ impl DeobfuscationEngine {
     /// * Deobfuscation Key
     /// * Encrypted UDS
     /// * Encrypted Field entropy
-    pub fn clear_secrets(&mut self) -> CaliptraResult<()> {
+    pub fn clear_secrets(&mut self) {
```

Comment:
Should this be checking if the DOE status bit [DEOBF_SECRETS_CLEARED](https://chipsalliance.github.io/caliptra-rtl/main/internal-regs/?p=clp.doe_reg.DOE_STATUS#DEOBF_SECRETS_CLEARED.desc) ever gets set?

I'm not sure what it would imply if that bit doesn't get set, but it makes me nervous that the bit exists but we aren't checking it.

Reply:
Unless this is just best effort for FIPS zeroization, in which case it's probably fine.

Reply:
Best practice is to check instead of making it void. Since we actually have a bit in hardware that gets set if secrets were cleared. Assumption is that if there is fault injection this bit won't be set and can be caught by simple check in the ROM to act on it. e.g. reset or hang.